src/main/java/core/basesyntax/StorageImpl.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax;
+
+public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_LENGTH_ARRAY = 10;
+    private K[] keysWithStorage;
+    private V[] valueWithStorage;
+    private int lengthStorage = 0;
```

Comment:
```suggestion
    private K[] keys;
    private V[] value;
    private int currentLength;
```
the naming can be a little bit more consise
and we don't need to set 0, this is a default value